rocky/rocky/views/organization_list.py

```diff
@@ -18,3 +28,32 @@ def get_queryset(self) -> list[Organization]:
             .prefetch_related("tags")
             .filter(id__in=[organization.id for organization in user.organizations])
         )
+
+    def post(self, request: HttpRequest, *args, **kwargs) -> HttpResponse:
+        """Perform actions based on action type"""
+        if request.POST.get("action") == "recalculate":
+            if not self.request.user.has_perm("tools.can_recalculate_bits"):
+                raise PermissionDenied()
+            organizations = self.request.user.organizations
+            number_of_bits = 0
+            failed = []
+            start_time = datetime.now()
+            for organization in organizations:
+                try:
+                    number_of_bits += OctopoesAPIConnector(settings.OCTOPOES_API, organization.code).recalculate_bits()
+                except Exception as exc:
+                    failed += [f"{organization}, ({str(exc)})"]
+                    logging.warning("Failed recalculating bits for %s, %s", organization, exc)
+            duration = datetime.now() - start_time
+            message = f"Recalculated {number_of_bits} bits for {len(organizations)-len(failed)} organizations."
+            message += f" Duration: {duration}."
+            if failed:
+                message += f"\nFailed for {len(failed)} organisations: {', '.join(failed)}"
+            messages.add_message(
+                request,
+                messages.INFO,
+                _(message),
+            )
+            return self.get(request, *args, **kwargs)
```

Comment:
Is this necessary? I believe the default behaviour of `ListView` is to render the template regardless of the HTTP verb being used

Reply:
![image](https://github.com/user-attachments/assets/21a63250-a3e9-44cc-a9c8-bb1a684a6eff)
 Well, removing it altogether is not an option